mungegithub/mungers/release-note-label.go

```diff
@@ -144,20 +144,15 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 		return
 	}
 
+	releaseNoteLabelNeededAdded := false
 	if !obj.HasLabel(releaseNoteLabelNeeded) {
 		obj.AddLabel(releaseNoteLabelNeeded)
+		releaseNoteLabelNeededAdded = true
 	}
 
-	if !obj.HasLabel(lgtmLabel) {
-		return
+	if releaseNoteLabelNeededAdded && obj.HasLabel(lgtmLabel) {
+		obj.WriteComment(releaseNoteBody)
 	}
-
-	if obj.HasLabel(doNotMergeLabel) {
-		return
-	}
-
-	obj.WriteComment(releaseNoteBody)
-	obj.AddLabel(doNotMergeLabel)
```

Comment:
If I only remove `obj.AddLabel(doNotMergeLabel)` then if just 4 lines above:
```
if obj.HasLabel(doNotMergeLabel) {
   return
}
```

Will not cause return, so we will be adding `obj.WriteComment(releaseNoteBody)` comment every time.
This is why I've added code to ensure that we're adding this comment only when we also added `obj.AddLabel(releaseNoteLabelNeeded)` at the same time.